### PR TITLE
Created release notes for 6.8.6

### PR DESCRIPTION
**Pull request by andsel**

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -37,6 +38,63 @@ This section summarizes the changes in the following releases:
 * <<logstash-6-1-1,Logstash 6.1.1>>
 * <<logstash-6-1-0,Logstash 6.1.0>>
 
+[[logstash-6-8-6]]
+* bump version to 6.8.6 https://github.com/elastic/logstash/pull/11327[#11327]
```
Comment (robbavey) [inline docs/static/releasenotes.asciidoc]: No need for this entry, I don't think

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -37,6 +38,63 @@ This section summarizes the changes in the following releases:
...
+*Jdbc Input*
+
+* Fixed issue where paging settings in configuration were not being honored [#361](https://github.com/logstash-plugins/logstash-input-jdbc/pull/361)
+
+*Rabbitmq Input*
+
+* Docs: Optional queue args: link to RabbitMQ docs instead
+
+*Snmp Input*
+
```
Comment (robbavey) [inline docs/static/releasenotes.asciidoc]: @yaauie Was pushing a minor update to this plugin for the patch release intentional? https://github.com/elastic/logstash/blob/6.8/Gemfile.jruby-2.5.lock.release#L375

**Comment by karenzone** [comment]
@elasticmachine, run elasticsearch-ci/docs

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -37,6 +38,63 @@ This section summarizes the changes in the following releases:
 * <<logstash-6-1-1,Logstash 6.1.1>>
 * <<logstash-6-1-0,Logstash 6.1.0>>
 
+[[logstash-6-8-6]]
```
Comment (robbavey) [inline docs/static/releasenotes.asciidoc]: Missing header
```suggestion
=== Logstash 6.8.6 Release Notes
```

**Comment by karenzone** [comment]
@andsel Thank you so much for generating the release notes. I will make some fixes that will get our docs-ci to green.

**Comment by karenzone** [comment]
@andsel  I took your commit and added to it in https://github.com/elastic/logstash/pull/11446

**Comment by andsel** [comment]
superseded by https://github.com/elastic/logstash/pull/11446

**Pull request closed without merging**